### PR TITLE
added firestation, policestation, hospital graphical representation

### PR DESCRIPTION
**Pull request by Janicekwing**

### CODE REVIEW

File: 01-python/safety/hospital.py [hunk 1]
```diff
@@ -1,3 +1,20 @@
-def draw_hospital():
-    print("hospital not found")
+def draw_hospital(length=20):
```
Comment (gallettilance) [inline 01-python/safety/hospital.py]: Please add docstrings to your functions

File: 01-python/safety/firestation.py [hunk 2]
```diff
@@ -1,3 +1,23 @@
-def draw_firestation():
-    print("fire station not found")
+def draw_firestation(length=20):
+    """
+        Argument length is the width of the building
```
Comment (gallettilance) [inline 01-python/safety/firestation.py]: why not call it `width` then?

File: 01-python/safety/firestation.py [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+        Returns an image of a firestation building using keyboard characters
```
Comment (gallettilance) [inline 01-python/safety/firestation.py]:
```suggestion
        Prints a firestation building using keyboard characters
```

File: 01-python/safety/firestation.py [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+    """
+
```
Comment (gallettilance) [inline 01-python/safety/firestation.py]: what if the input is invalid? ex: < 0 etc.

File: 01-python/safety/hospital.py [hunk 5]
```diff
@@ -1,3 +1,23 @@
...
+    """
+        Argument length is the width of the building
```
Comment (gallettilance) [inline 01-python/safety/hospital.py]: Same as above - why not call is width

File: 01-python/safety/hospital.py [hunk 6]
```diff
@@ -1,3 +1,23 @@
...
+        Returns an image of a hospital building using keyboard characters
```
Comment (gallettilance) [inline 01-python/safety/hospital.py]: same as above

**Comment by gallettilance** [comment]
@Janicekwing Should I close this PR or are you still planning to work on it?

**Comment by Janicekwing** [comment]
No you can delete that.

Thanks 
Janice

**Pull request closed without merging**